rr/gstreamer/mediamanager.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
```

Comment:
Please discuss with @Solzc what module naming convention we are going to adopt. I believe @Solzc is using `snake_case`.